bootstrap/helpers/docker.php

```
@@ -1093,19 +1093,18 @@ function getContainerLogs(Server $server, string $container_id, int $lines = 100
 {
     if ($server->isSwarm()) {
         $output = instant_remote_process([
-            "docker service logs -n {$lines} {$container_id}",
+            "docker service logs -n {$lines} {$container_id} 2>&1",
```

PHPMD (2.15.0)

The variable $container_id is not named in camelCase. (undefined)

(CamelCaseVariableName)